packages/core/overlays/core-overlays-browser-internal/src/flyout/flyout_service.tsx

```diff
@@ -10,7 +10,7 @@
 
 import { EuiFlyout } from '@elastic/eui';
 import React from 'react';
-import { render, unmountComponentAtNode } from 'react-dom';
+import * as ReactDOM from 'react-dom';
```

Comment:
nit: we shouldn't import as `*` in Kibana.  We're considering an `eslint` rule to disallow it.  Mostly it's to get in the habit so we don't bloat plugins and packages.